optuna/study.py

```diff
@@ -4,7 +4,15 @@
 import math
 import multiprocessing
 import multiprocessing.pool
-import pandas as pd
+
+try:
+    import pandas as pd  # NOQA
+    _available = True
```

Comment:
I prefer the more specific variable name like `_pandas_available` to avoid conflicts in the module namespace. And I think following code is enough for this purposes:

```python
try:
    import pandas
except ImportError as e:
    pandas = None


def _check_pandas_availability():
    # type: () -> None

     if pandas is None:
         raise ImportError(...)
```

Reply:
thanks!